laravel-app/app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -50,9 +51,14 @@ public function __construct()
     protected function validator(array $data)
     {
         return Validator::make($data, [
```

Comment:
Do not expose your validation requests in your controller. You should use **form request** in laravel